docs/src/components/Layout/Page/Page.component.tsx

```diff
@@ -255,20 +275,20 @@ export const Page = ({
                                 />
                             </HamburgerCol>
 
-                            <CenteredCol md={2} xs={2} sm={4}>
+                            <LogoCol md={2} lg={3} xl={2}>
                                 <StyledLogoContainer to="/">
                                     <img
                                         alt="Anchor Logo Horizontal"
                                         src={logo}
                                     />
                                 </StyledLogoContainer>
-                            </CenteredCol>
+                            </LogoCol>
 
-                            <CenteredCol md={2} xs={1} sm={1}>
-                                {/* TODO: Search will go here */}
-                            </CenteredCol>
+                            <SearchCol>
+                                <SearchInput />
```

Comment:
When I select a result and go to the linked page, the Search Input doesn't seem to exist anymore..

Reply:
Ah, just realized thats because the links when I run it locally are pointing at the live prod site.